setup.cfg

```diff
@@ -0,0 +1,42 @@
+[metadata]
+name = cypari2
+version = file: VERSION
+long_description = file: README.rst
+description = A Python interface to the number theory library PARI/GP
+author= Luca De Feo, Vincent Delecroix, Jeroen Demeyer, Vincent Klein
+author_email= sage-devel@googlegroups.com
+url= https://github.com/sagemath/cypari2
+download_url = https://pypi.org/project/cypari2/#files
+license= GNU General Public License, version 2 or later
+platforms = any
+classifiers =
+    License :: OSI Approved :: GNU General Public License v3 (GPLv3)
+    Programming Language :: C++
+    Programming Language :: Python
+    Development Status :: 5 - Production/Stable
+    Operating System :: Unix
+    Intended Audience :: Science/Research
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.4
+    Programming Language :: Python :: 3.5
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+keywords =
+    PARI/GP
+    number theory
+
+[options]
+packages =
+    cypari2
+setup_requires =
+    Cython>=0.29
```

Comment:
setup_requires is no longer a thing. build deps are declared in pyproject.toml

Reply:
ha, right.

Reply:
Hopefully better with 013925f ?